client/app.jsx

```diff
@@ -31,20 +32,26 @@ export default class App extends React.Component {
     if (route.path === 'New-Date') {
       return <NewDateForm listId={route.params.get('listId')} />;
     }
-    if (route.path === 'Lists') {
+    if (route.path === 'Read-List') {
       return <ListDetails listId={route.params.get('listId')} />;
     }
+    if (route.path === 'Generate-Date') {
+      return <GenerateDate />;
+    }
   }
 
   render() {
     const { path } = this.state.route;
+    const withNav = ['New-List', 'New-Date'].includes(path);
+    let hasNav = 'with-navbar';
+    withNav ? hasNav = '' : hasNav = 'with-navbar';
```

Comment:
Ok, now I see what you're doing here. We can finish this out like this:

```js
const pageClass = withNav ? 'page with-navbar' : 'page';
```

then just pass that

```jsx
<div className={pageClass}>
```